tests/trainer/test_trainer.py

```diff
@@ -1285,6 +1286,7 @@ def test_resume_training_with_randomness(self):
             self.assertAlmostEqual(b, b1, delta=1e-5)
 
     @slow
+    @require_accelerate
```

Comment:
This test requires `accelerate`, which is installed in daily CI image, but not past CI images.

(So far the past CI avoids installing many other dependencies, but this could be changed in the future)